tools/msvs/install_tools/install_tools.bat

```diff
@@ -20,8 +22,15 @@ echo.
 echo You can close this window to stop now. This script can be invoked from the
 echo Start menu. Detailed instructions to install these tools manually are
 echo available at https://github.com/nodejs/node-gyp#on-windows
-echo.
-pause
+:acceptretry
+echo.
+echo Your computer may REBOOT SEVERAL TIMES WITHOUT FURTHER WARNING.
+echo Please type YES followed by enter to confirm that you have saved all your
+set /p "ACCEPT_PROMPT=work and closed all open programs: "
+if /i not "%ACCEPT_PROMPT%"=="yes" (
+  echo Please type YES or close the window to exit.
```

Comment:
`Please type YES or close the window to exit.` reads like typing Yes or closing the window will exit. I think the `type Yes` bit should read more like the initial prompt to `type YES to confirm or...`.